src/plugins/i18n.ts

```diff
@@ -19,69 +19,59 @@ interface LoadingFontFaceProperty {
   only?: Array<string>
 }
 
-const unicodeHalfAndFullWidthForms = [
-  "U+FF00-FFEF"
-];
-
-const unicodeCJK = [
-  "U+2E80-2EFF",
-  "U+3000-303F",
-  "U+31C0-31EF",
-  "U+3200-32FF",
-  "U+3400-4DBF",
-  "U+F900-FAFF",
-  "U+FE30-FE4F",
-].join(",");
-
-const unicodeHangul = [
-  "U+1100-11FF",
-  "U+3130-318F",
-  "U+A960-A97F",
-  "U+AC00-D7AF",
-  "U+D7B0-D7FF",
-].join(",");
-
-const unicodeGana = "U+3040-30FF";
-
-// Common letters both Japanese and Chinese
-const unifiedIdeographsCJK = "U+4E00-9FFF";
+const unicodeRanges = {
+  fullwidth: "U+FF00-FFEF",
+  hangul: "U+1100-11FF,U+3130-318F,U+A960-A97F,U+AC00-D7AF,U+D7B0-D7FF",
+  kana: "U+3040-30FF",
+  CJKCommon: "U+2E80-2EFF,U+3000-303F,U+31C0-31EF,U+3200-32FF,U+3400-4DBF,U+F900-FAFF,U+FE30-FE4F",
+  CJKIdeograph: "U+4E00-9FFF",
+};
+const rangesByLanguage = {
+  korean: [unicodeRanges.CJKCommon, unicodeRanges.hangul].join(","),
+  chinese: [unicodeRanges.CJKCommon, unicodeRanges.fullwidth, unicodeRanges.CJKIdeograph].join(","),
+  japanese: [unicodeRanges.CJKCommon, unicodeRanges.fullwidth, unicodeRanges.kana, unicodeRanges.CJKIdeograph].join(",")
+};
 
 const fonts: Array<LoadingFontFaceProperty> = [
   // korean
-  { face: new FontFace("emerald", "url(./fonts/PokePT_Wansung.ttf)", { unicodeRange: unicodeHangul }) },
   {
-    face: new FontFace("pkmnems", "url(./fonts/PokePT_Wansung.ttf)", { unicodeRange: unicodeHangul }),
+    face: new FontFace("emerald", "url(./fonts/PokePT_Wansung.woff2)", { unicodeRange: rangesByLanguage.korean }),
```

Comment:
```suggestion
    face: new FontFace("emerald", "url(./fonts/PokePT_Wansung.woff2), url(./fonts/PokePT_Wansung.ttf)", { unicodeRange: rangesByLanguage.korean }),
```
호환성을 위한 여러 파일 로딩 시도나 정리를 위한 기존 .ttf파일 삭제가 필요해보입니다.

Reply:
삭제하는 방향으로 가겠습니다!